### PR TITLE
E_CLASSROOM-226 [Bugfix] Remove Friend's Score UI If There's No Friends Score

### PR DESCRIPTION
**Pull request by Teraaay**
### Issue Link

https://framgiaph.backlog.com/view/E_CLASSROOM-226

### Definition of Done
- [x] There should be no Friend's Score section in the Quiz Results page when there are no friend's scores.

### Commands to Run
N/A

#### Main note
- N/A
#### Pages Affected
- http://localhost:3003/categories/7/quizzes/7/questions

### Scenarios/ Test Cases
- [x] There should be `no Friend's Score section` when `the friend's score list is empty` 

### Screenshots
image

### CODE REVIEW

**Comment by catampongan** [comment]
### Working
- [x] Able to remove UI if friends not taking the quiz yet or no friends at all

### Comment
- Align the content of friend's score (refer from figma)

------

image

Screenshot 2022-01-26 141037

`On Figma`

image


------
- Total Test Time: 0.25 hr

**Comment by catampongan** [comment]
## Approved PR #79 
- [x] Able to remove UI if friends not taking the quiz yet or no friends at all
- [x] Able to replicate alignment of friend's score in Figma

--------------
image
Screenshot 2022-01-26 141442

------

- Total Test Time: 0.5 hr
- Total Iterations: 1
- Total Comments: 1

File: src/pages/student/quizzes/QuizResult/index.js [hunk 1]
```diff
@@ -72,73 +71,72 @@ const QuizResult = ({ score, total, quizId, categoryId }) => {
...
-                                    <img
-                                      className={style.ResultsizeOfAvatar}
-                                      alt="avatar"
-                                      src={friendScore.avatar}
-                                    />
-                                  </div>
-                                )}
-                              </td>
-                              <td className={style.friendsName}>
+                {friendsScore?.length > 0 ? (
```
Comment (karlolee50) [inline src/pages/student/quizzes/QuizResult/index.js:74]: should friendsScore have a question mark?

File: src/pages/student/quizzes/QuizResult/index.js [hunk 2]
```diff
@@ -72,73 +71,72 @@ const QuizResult = ({ score, total, quizId, categoryId }) => {
...
+                              className={style.friendsScoreInfo}
+                            >
+                              <div>
+                                <img
+                                  className={style.friendsAvatar}
+                                  alt="avatar"
+                                  src={
+                                    friendScore.avatar === null
+                                      ? 'https://www.pngall.com/wp-content/uploads/5/Profile-Avatar-PNG.png'
+                                      : friendScore.avatar
```
Comment (karlolee50) [inline src/pages/student/quizzes/QuizResult/index.js]:
```suggestion
                                  src={
                                    friendScore.avatar || 'https://www.pngall.com/wp-content/uploads/5/Profile-Avatar-PNG.png'
```

**Review by karlolee50 — APPROVED** [review]
Review time: 15 minutes
Comment/s: 1

**Pull request merged**